Makefile

```diff
@@ -243,7 +243,7 @@ release:
 	 echo "  git push origin $$X.$$Y $$X.$$Y.$$Z"
 
 shell: $(GOBUILDER)
-	@$(DOCKER_GO) bash $(GOTREE) $(GOMODULE)
+	@DOCKER_GO_ARGS=-t ; $(DOCKER_GO) bash $(GOTREE) $(GOMODULE)
```

Comment:
Sure you want a semi-colon? You're just setting the DOCKER_GO_ARGS in the env, right?

Reply:
Oops. I missed that one in review....

Reply:
Semicolon is needed since DOCKER_GO is a shell function actually, not a command. FOO=bar function foo() syntax doesn't work